### DISCUSSION TITLE
Suggestion for Mute Pluging

### DISCUSSION THREAD

**Discussion by illiaKuryaki** [ChatAdditions/ChatsAdditions_AMXX#146] [Ideas]
hi, on my community we have some people who just will not stop talking xD, so a few of us, including myself, we use the mute all, but the plugin notifices everyone else about us muting everyone, will be nice to have the option to disable that, and be silence if someone mute all or someone.

Thanks for your work !

**Comment by wopox1337** [comment]
Thank you for your suggestion, I'll think about adding functionality for that.

At this point, you can comment out these lines and the message will not appear.

comment this:
https://github.com/ChatAdditions/ChatAdditions_AMXX/blob/master/cstrike/addons/amxmodx/scripting/CA_Mute.sma#L169-L171
and this https://github.com/ChatAdditions/ChatAdditions_AMXX/blob/master/cstrike/addons/amxmodx/scripting/CA_Mute.sma#L194-L196

**Reply by illiaKuryaki** [reply]
Rgr, thanks !
